prac2-pages/chhotu/lib/main.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MaterialApp(
+      home: Project(),
+    ));
+
+class Project extends StatefulWidget {
+  const Project({super.key});
+
+  @override
+  State<Project> createState() => _ProjectState();
+}
+
+class _ProjectState extends State<Project> {
+  bool value = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: <Widget>[
+          Container(
```

Comment:
There is a major issue with your UI here, as it leads to an overflow at the bottom.

- Whenever we click on a textfield and the keypad opens up, there is a further overflow at the bottom.
- There's too much whitespace near the top.

**Possible fixes**
- Might want to use ScreenUtil package for creating pixel-perfect and adaptive UI. Research on it
- Learn the use of Columns and Rows, and you can use them to align items 

**Screenshot** 
![WhatsApp Image 2022-12-23 at 12 15 16](https://user-images.githubusercontent.com/106133014/209286490-cd3678b7-8be4-4215-8f89-6fc137b0257b.jpeg)